tracing-honeycomb/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tracing-honeycomb"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
fwiw 0.2 isn't even out yet, see https://github.com/inanna-malick/tracing-honeycomb/pull/5

Reply:
0.2 is out, I just never published it (sorry, 2020 has been.. very 2020 recently) https://github.com/inanna-malick/tracing-honeycomb/blob/master/tracing-honeycomb/Cargo.toml#L3